### ISSUE TITLE
Check server connection on POP3 always disconnected

### ISSUE THREAD

**Issue by HandyGuySoftware** [HandyGuySoftware/dupReport#107]
Does it do this for IMAP as well?

```
dremail.processNextMessage()
EmailServer.Connect(protocol=[pop3] address=[pop.gmail.com] port=[995] account=[backups@friedfamily.net] passwd=[flartsnoggle301] encryption=[tls] keepalive=[True] folder=[Inbox])
server=<poplib.POP3_SSL object at 0x0000018EA594E1D0> keepalive=True
Cheeking server connection
Server pop.gmail.com timed out. Reconnecting.
EmailServer.Connect(protocol=[pop3] address=[pop.gmail.com] port=[995] account=[backups@friedfamily.net] passwd=[flartsnoggle301] encryption=[tls] keepalive=[True] folder=[Inbox])
server=None keepalive=True
Initial connect using POP3
Logged in. retVal=b'+OK send PASS'
Entered password. retVal=b'+OK Welcome.'
```

**Comment by HandyGuySoftware** [comment]
Does not happen for IMAP server.

**Comment by HandyGuySoftware** [comment]
Fixed in 2.2.5

**Comment by HandyGuySoftware** [comment]
Fixed in 2.2.5
